control/timeresp.py

```diff
@@ -210,40 +210,45 @@ def forced_response(sys, T=None, U=0., X0=0., transpose=False,
 
     T : array_like, optional for discrete LTI `sys`
         Time steps at which the input is defined; values must be evenly spaced.
+        If None, `U` must be given and and `len(U)` time steps of sys.dt are
+        simulated. If sys.dt is None or True (undetermined time step), a dt
+        of 1.0 is assumed.
 
     U : array_like or float, optional
-        Input array giving input at each time `T` (default = 0).
+        Input array giving input at each time `T`
 
         If `U` is ``None`` or ``0``, a special algorithm is used. This special
         algorithm is faster than the general algorithm, which is used
         otherwise.
 
     X0 : array_like or float, optional
-        Initial condition (default = 0).
+        Initial condition.
 
     transpose : bool, optional
         If True, transpose all input and output arrays (for backward
-        compatibility with MATLAB and :func:`scipy.signal.lsim`).  Default
-        value is False.
+        compatibility with MATLAB and :func:`scipy.signal.lsim`).
 
-    interpolate : bool, optional (default=False)
+    interpolate : bool, optional
         If True and system is a discrete time system, the input will
         be interpolated between the given time steps and the output
         will be given at system sampling rate.  Otherwise, only return
         the output at the times given in `T`.  No effect on continuous
-        time simulations (default = False).
+        time simulations.
 
     return_x : bool, optional
-        If True (default), return the the state vector.  Set to False to
-        return only the time and output vectors.
+        - If False, return only the time and output vectors.
+        - If True, also return the the state vector.
+        - If None, determine the returned variables by
+          config.defaults['forced_response.return_x'], which was True
+          before version 0.9 and is False since then.
```

Comment:
Since this docstring is only visible in version 0.9+, is there a reason to give the pre-0.9 value?

Reply:
I don't see any harm in giving some hints on history. If old user code starts to fail because the behavior changed, this helps a lot to see why.